### DISCUSSION TITLE
Correct way to get default date format for a locale

### DISCUSSION THREAD

**Discussion by KonradLinkowski** [js-fns/date-fns#3100] [Q&A]
Is using `locale.formatLong?.date({ width: 'short' })` a valid way of getting default date format or is it a hack? Because it's not documented, but it works.

Or is there a better way to do this?

**Comment by Kaspanoombro** [comment]
Hello, i am trying to get the same. I am using angular 14. I have everything working but i could not user your code:
locale.formatLong?.date({ width: 'short' })
Can you help me to have the "locale" working? Thank you.
